---
name: Jaime A Castro-Mondragon
position: Bioinformatician
address: Oslo, Norway
www: jaimicore.github.io
email: jamondra@uio.no
linkedin: jaime-castro-mondragon
bluesky: jaimicore
github: jaimicore
date: "`r format(Sys.time(), '%B %Y')`"
profilepic: data/Jaime.jpg
headcolor: 0570b0
aboutme: "I am a computational biologist with 11 years of experience in research, sequence analysis, methods development and data visualisation. I have worked in both the academic and private sector. My main expertise is the development of algorithms to analyze multivariate data and apply them in a wider range of biomedical fields such as vaccine design or gene expression."
output: 
  vitae::awesomecv:
    keep_tex: false
    show_footer: true
    includes:
      in_header: header.txt
  pdf_document:
    latex_engine: xelatex
always_allow_html: yes
editor_options: 
  chunk_output_type: console
header-includes:
    - \usepackage{xcolor}
    - \definecolor{mycolor}{RGB}{255, 87, 51}  # Custom orange color
    - \usepackage{hyperref}
    - \hypersetup{colorlinks=true, linkcolor=mycolor, urlcolor=mycolor}
---


```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, warning = FALSE, message = FALSE)
library(vitae)
library(readr)
library(lubridate)
library(dplyr)
library(here)
library(readxl)
library(glue)
library(tidyr)
library(kableExtra)
library(purrr)
library(stringr)

show.publications <- TRUE
publication.short <- !show.publications

```


```{r data, inclue=FALSE}
jobs.raw <- read_csv(here::here("data", "CV_events.csv"))
jobs <-
  jobs.raw %>%
  fill(Start, End, What, With, Where, Tag) %>%
  # dplyr::mutate(End = ifelse(End == "Present", yes = dmy(as.Date(today())), no = End)) %>% 
  dplyr::mutate(Start = dmy(Start),
                End   = dmy(End)) %>%
  dplyr::mutate(When  = case_when(Start == End ~ glue("{year(Start)} - Present") %>% as.character(),
                                  year(Start) == year(End) ~ year(End) %>% as.character(),
                                  TRUE ~ glue("{year(Start)} - {year(End)}") %>% as.character()
  ))
```

# Working Experience

\renewcommand{\arraystretch}{0.85}
```{r research_work}


jobs %>%
  dplyr::filter(Tag == "Work/Research") %>%
  arrange(desc(Start)) %>%
  detailed_entries(
    what  = What,
    when  = When,
    with  = With,
    where = Where,
    why   = map(Why, ~ str_split(.x, "\\.\\s*")[[1]] %>% 
                 discard(~ .x == "") %>%
                 str_replace_all("JASPAR database", "\\\\href{https://jaspar.uio.no/}{JASPAR database}") %>% 
                 str_replace_all("added motif clusters to the website", "\\\\href{https://jaspar.uio.no/matrix-clusters/nematodes/}{added motif clusters to the website}") %>% 
                 str_replace_all("integration method", "\\\\href{https://bitbucket.org/CBGR/dysmir_pipeline/}{integration method}") %>% 
                 str_replace_all("Anthony Matelier", "\\\\href{https://mathelierlab.com/}{Anthony Matelier}") %>% 
                 str_replace_all("motif analysis pipeline", "\\\\href{https://bitbucket.org/CBGR/jaspar_curation_pipeline}{motif analysis pipeline}") %>% 
                 paste0(., ".")),
    .protect = FALSE
  )

```

<!-- \hspace{0.75cm} -->

# Toolkit

\renewcommand{\arraystretch}{0.95}
```{r Comp_skills}

comp.skills <- jobs %>% 
                dplyr::filter(Tag == "Comp_skills") %>% 
                # detailed_entries(with = With,
                #                  what = What) %>% 
                dplyr::select(What, With, Why) %>% 
                separate(Why, sep = ";", into = c("Category", "Description")) %>% 
                dplyr::select(!Description) %>% 
                rename(Level = With) %>% 
                group_by(Category) %>%
                mutate(Category = if_else(row_number() == 1, Category, "")) %>%
                ungroup()

# comp.skills <- comp.skills[1:13,]

# Define color mapping for levels
color_mapping <- c("Proficient" = "#08306b", 
                   "Intermediate" = "#2171b5", 
                   "Beginner" = "#6baed6")  # Changed 3rd color for uniqueness

# Process the table
formatted_tbl <- comp.skills %>%
  mutate(Color = color_mapping[Level],  # Assign colors dynamically
         Level = cell_spec(Level, "latex", color = Color)) %>%
  select(-Color)  #

# Create the table

formatted_tbl %>%
  kable(booktabs = T, align = 'l', escape = F, format = "latex", linesep = "") %>%
  kable_styling(full_width = F, font_size = 9, latex_options = c("HOLD_position", "scale_down"), position = "float_left") %>%
  column_spec(column = 1:3, width = c('1.75cm', '1.75cm', '3.25cm'), background = c("#FFFFFF", "#deebf7")) %>%
  row_spec(0, bold = TRUE, background = c("#FFFFFF"), align = "left")



```


\hspace{0.5cm}
\footnotesize \textbf{Technical Skills}: clustering, dimensionality reduction, exploratory data analysis (EDA), data wrangling, multivariate analysis, statistical tests, Markov chains, ML (Random Forest, Linear Regression, Logistic Regression, model evaluation), Deep Learning (neural networks), Data Curation, pattern discovery, pattern matching.

\hspace{0.5cm}
\footnotesize \textbf{Tools}: jupyter notebooks, Rstudio, emacs, VS code, google docs, Linux (Ubuntu).

\clearpage
\hspace{0.5cm}

# Education

```{r education}
jobs %>% 
  dplyr::filter(Tag == "Education") %>% 
  arrange(desc(Start)) %>% 
  detailed_entries(what  = What,
                   when  = When,
                   with  = With,
                   where = Where,
                   why   = Why)
```


\hspace{1cm}

# Languages

```{r Languages, results='markup'}

# Adapted from: https://github.com/loreabad6/R-CV/blob/master/CV.Rmd
# jobs %>%
#   dplyr::filter(Tag == "Languages") %>%
#   detailed_entries(what = What,
#                    why  = Why)

data.frame(
  Spanish = cell_spec(c("Native"), 'latex', color = '#005c7b'),
  English = cell_spec(c("Fluent"), 'latex', color = '#009acd'),
  French = cell_spec(c("Fluent"), 'latex', color = '#009acd'),
  Norwegian = cell_spec(c("Intermediate (A2)"), 'latex', color = '#4cb8dc')) |> 
  kable(booktabs = TRUE, align = 'c', escape = FALSE, format = "latex", linesep = "") |>
  kable_styling(full_width = T, font_size = 10, latex_options = c("HOLD_position", "scale_down")) |>
  column_spec(column = 1:4, width = '3cm')


```


<!-- # Awards and Honors -->

<!-- ```{r awards} -->
<!-- jobs %>%  -->
<!--   dplyr::filter(Tag == "Honors_prices") %>%  -->
<!--   arrange(desc(Start)) %>%  -->
<!--   detailed_entries(what = What, -->
<!--                    when = When) -->
<!-- ``` -->


<!-- # Teaching Experience -->

<!-- ```{r teaching} -->
<!-- jobs %>% -->
<!--   dplyr::filter(Tag == "Teaching") %>% -->
<!--   arrange(desc(Start)) %>%  -->
<!--   detailed_entries(what  = What, -->
<!--                    when  = When, -->
<!--                    with  = With, -->
<!--                    where = Where, -->
<!--                    why   = Why) -->
<!-- ``` -->


\hspace{1cm}

```{asis, echo=show.publications, eval=show.publications}

# Publications

\hspace{1cm}

See the complete list of publications (including preprints) and citations in my google scholar profile: [*pcevKk0AAAA*](https://scholar.google.fr/citations?user=pcevKk0AAAAJ&hl=en)

\hspace{1cm}

*ORCID* profile: [0000-0003-4069-357X](https://orcid.org/0000-0003-4069-357X)

\hspace{1cm}

\* = equal contributions (first-author)

^ = equal contributions (second-author)

\# = co-corresponding authors

\hspace{1cm}
## Peer-reviewed

1. Baydar DO*, Rauluseviciute I*, Aronsen DR^, Blanc-Mathieu R^, Bonthuis I^, de Beukelaer H^, Ferenc K^, Jegou A^, Kumar V^, Lemma RB^, Lucas J^, Pochon M^, Yun CM^, Ramalingam V^, Deshpande SS^, Patel A, Marinov GK, Wang AT, Aguirre A, **Castro-Mondragon JA**, Baranasic D, Chèneby J, Gundersen S, Johansen M, Khan A, Kuijjer ML, Hovig E, Lenhard B#, Sandelin A#, Vandepoele K, Wasserman WW#, Parcy F#, Kundaje A#, Mathelier A#. *JASPAR 2026: expansion of transcription factor binding profiles and integration of deep learning models.* Nucleic Acids Research (2025)

2. Taboada-Castro H, Hernandez-Alvarez AJ, **Castro-Mondragon JA**, and Encarnacion-Guervara S. *RhizoBindingSites v2.0 Is a Bioinformatic Database of DNA Motifs Potentially Involved in Transcriptional Regulation Deduced From Their Genomic Sites*. Bioinformatics and Biology Insights (2024)

3. Rauluseviciute I, Launay T, Barzaghi G, Nikumbh S, Lenhard B, Krebs AR, **Castro-Mondragon JA**, and Mathelier A. *Identification of transcription factor co-binding patterns with non-negative matrix factorization.* Nucleic Acids Research (2024)

4. Rauluseviciute I\*, Riudavets-Puig R\*, Blanc-Mathieu R^, **Castro-Mondragon JA**^, Ferenc K^, Kumar V^, Berhanu Lemma R^, Lucas J^, Cheneby J, Baranasic D, Khan A, Fornes O, Gundersen S, Johansen M, Hovig E, Lenhard B#, Sandelin A#, Wasserman WW#, Parcy F#, and Mathelier A#. *JASPAR 2024: 20th anniversary of the open-access database of transcription factor binding profiles.* Nucleic Acids Research (2024)

5. **Castro-Mondragon JA**, Ragle Aure M, Lingjærde OC, Langerød A, Martens JWM, Børresen-Dale AL, Kristensen VN, and Mathelier A. *Cis-regulatory mutations associate with transcriptional and post-transcriptional deregulation of gene regulatory programs in cancers*. Nucleic Acids Research (2022)

6. Saha S\*, Spinelli L\*, **Castro-Mondragon JA**, Kervadec A,  Lynott M, Kremmer L, Roder L, Sallouha J, Torres M, Brun C, Vogler G, Bodmer R, Colas AR#, Ocorr K#, and Perrin L#. *Genetic architecture of natural variations of cardiac performance in flies*. eLife (2022)

7. Santana-Garcia W\*, **Castro-Mondragon JA**\*,  Padilla-Galvez M, Nguyen NTT*, Elizondo-Salas A, Ksouri N, Gerbes F, Thieffry D, Vincens P, Contreras-Moreira B#, van Helden J#, Thomas-Chollier M#, Medina-Rivera A#. *RSAT 2022: regulatory sequence analysis tools*. Nucleic Acids Research (2022)

8. **Castro-Mondragon JA**\*, Riudavets-Puig R\*, Rauluseviciute I\*, Berhanu Lemma R, Turchi L, Blanc-Mathieu R, Lucas J, Boddie P, Khan A, Manosalva Perez N, Fornes O, Leung TY, Aguirre A, Hammal F, Schmelter D, Baranasic D, Ballester B, Sandelin A#, Lenhard B#, Vandepoele K, Wasserman WW#, Parcy F#, and Mathelier A#. *JASPAR 2022: the 9th release of the open-access database of transcription factor binding profiles*. Nucleic Acids Research (2021)

9. Santiago-Algarra D, Souaid C, Singh H, Dao TML, Hussain S, Medina-Rivera A, Ramirez-Navarro L, **Castro-Mondragon JA**, Sadouni N, Charbonnier G, Spicuglia S. *Epromoters function as a hub to recruit key transcription factors required for the inflammatory response*. Nature Communications (2021)

10. Riudavets-Puig R, Boddie P, Khan A, **Castro-Mondragon JA**, and Mathelier A. *UniBind: maps of high-confidence direct TF-DNA interactions across nine species*. BMC Genomics (2021)

11. Ragle Aure M, Fleischer T, Bjørklund S, Ankill J, **Castro-Mondragon JA**, OSBREAC (Oslo Breast Cancer Research Consortium), Børresen-Dale AL, Tost J, Sahlberg KK, Mathelier A, Tekpli X#, Kristensen VN#. *Crosstalk between microRNA expression and DNA methylation drives the hormone-dependent phenotype of breast cancer*. Genome Medicine (2021)

12. Ksouri N, **Castro-Mondragon JA**, Montardit-Tarda F, van Helden J, Contreras-Moreira B#, and Gogorcena Y#. *Tuning promoter boundaries improves regulatory motif discovery in non-model plants: the peach example*. Plant Physiology (2021)

13. Taboada-Castro H, **Castro-Mondragon JA**, Aguilar-Vera A, Hernandez-Alvarez AJ, van Helden J, and Encarnacion-Guervara S. *RhizoBindingSites, a Database of DNA-Binding Motifs in Nitrogen-Fixing Bacteria Inferred Using a Footprint Discovery Approach*. Frontiers in Microbiology (2020)

14. Fornes O\*, **Castro-Mondragon JA**\*, Khan A\*, van der Lee Robin, Zhang X, Richmond PA, Modi BP, Correard S, Gheorghe M, Baranasic D, Santana-Garcia Walter, Tan G, Cheneby J, Ballester B, Parcy F, Sandelin A#, Lenhard B#, Wasserman WW#, and Mathelier A#. *JASPAR 2020: update of the open-access database of transcription factor binding profiles*. Nucleic Acids Research (2019)

15. Nguyen NTT\*, Contreras-Moreira B\*, **Castro-Mondragon JA**, Santana-Garcia W, Ossio R, Robles-Espinoza CD, Bahin M, Collombet S, Vincens P, Thieffry D, van Helden J#, Medina-Rivera A#, Thomas-Chollier M#. *RSAT 2018: regulatory sequence analysis tools 20th anniversary*. Nucleic Acids Research (2018)

16. Taboada H\*, Meneses N\*, Dunn MF\*, Vargas-Lagunas C, Buchs N, **Castro-Mondragon JA**, Heller M, and Encarnacion S. *Proteins in the periplasmic space and outer membrane vesicles of Rhizobium etli CE3 grown in minimal medium are largely distinct and change with growth phase*. Microbiology (2018)

17. Khan A\*, Fornes O\*, Stigliani A\*, Gheorghe M, **Castro-Mondragon JA**, van der Lee R, Bessy A, Chèneby J, Kulkarni S, Tan G, Baranasic D, Arenillas D, Sandelin A#, Vandepoele K, Lenhard B#, Ballester B, Wasserman W#, Parcy F, Mathelier A#. *JASPAR 2018: update of the open-access database of transcription factor binding profiles and its web framework*. Nucleic Acids Research (2017)

18. Dao LM\*, Galindo-Albarran AO\*, **Castro-Mondragon JA**^, Andireu-Soler C^, Medina-Rivera A^, Souadi C, Charbonnier G, Griffon A, Vanhille L, Stephen S, Alomairi J, Soler C, Stephen T, Martin D, Torres M, Fernandez N, Soler E, van Helden J, Puthier D, Spicuglia S. *Genome-wide characterization of mammalian promoters with distal enhancer functions*. Nature Genetics (2017)

19. **Castro-Mondragon JA**, Jaeger S, Thieffry D, Thomas-Chollier M#, and van Helden J#. *RSAT matrix-clustering: dynamic exploration and redundancy reduction of transcription factor binding motif collections*. Nucleic Acids Research (2017)

20. **Castro-Mondragon JA**\*, Rioualen C\*, Contreras-Moreira B, van Helden J. *RSAT::Plants: Motif Discovery in ChIP-Seq Peaks of Plant Genomes*. Plant Synthetic Promoters - Springer Protocol (2016)

21. Contreras-Moreira B#, **Castro-Mondragon JA**, Rioualen C, Cantalapiedra CP, van Helden J. *RSAT::Plants: Motif Discovery Within Clusters of Upstream Sequences in Plant Genomes*. Plant Synthetic Promoters - Springer Protocol (2016)

22. Gama-Castro S\*, Salgado H\*, Santos-Zavaleta A, Ledezma-Tejeida D, Muñiz-Rascado L, García-Sotelo JS, Alquicira-Hernández K, Martínez-Flores I, Pannier L, **Castro-Mondragon JA**, Medina-Rivera A, Solano-Lira H, Bonavides-Martínez C, Pérez-Rueda E, Alquicira-Hernández S, Porrón-Sotelo L, López-Fuentes A, Hernández-Koutoucheva A, Del Moral-Chávez V, Rinaldi F, Collado-Vides J. *RegulonDB version 9.0: high-level integration of gene regulation, coexpression, motif clustering and beyond*. Nucleic Acids Research (2015)

23. Medina-Rivera A\*, Defrance M\*, Sand O\*, Herrmann C, **Castro-Mondragon JA**, Delerce J, Jaeger S, Blanchet C, Vincens P, Caron C, Staines DM, Contreras-Moreira B, Artufel M, Charbonnier–Khamvongsa L, Hernandez C, Thieffry D, Thomas-Chollier M#, van Helden J#. *	RSAT 2015 : Regulatory Sequence Analysis Tools*. Nucleic Acids Research (2015)

24. Rogel MA, Bustos P, Santamaría RI, González V, Romero D, Miguel AC, Lozano L, **Castro-Mondragon JA**, Martínez-Romero J, Ormeño-Orrillo E, Martínez-Romero E. *Genomic basis of symbiovar mimosae in Rhizobium etli*. BMC Genomics (2014)

\hspace{1cm}

## Preprints

No preprints at this moment.
```


```{asis, echo=publication.short, eval=publication.short}

# Publications

23 Peer-reviewed publications (6 as first author).

See the complete list of publications (including preprints) and citations in my google scholar profile: [*pcevKk0AAAA*](https://scholar.google.fr/citations?user=pcevKk0AAAAJ&hl=en)

*ORCID* profile: [0000-0003-4069-357X](https://orcid.org/0000-0003-4069-357X)

```


\hspace{1cm}

# Reviewed manuscripts

*Web of Science (prev. Publons)* profile: [1499198](https://www.webofscience.com/wos/author/record/P-6297-2019)

+30 reviewed manuscripts in the following journals: Bioinformatics, Genetics, Molecular Plant, Nucleic Acids Research, Genome biology, Genome Biology and Evolution, Frontiers in Genetics, eLife.


<!-- # Presentations -->

<!-- ## Talks -->

<!-- ```{r presentations} -->
<!-- jobs %>% -->
<!--   dplyr::filter(Tag == "Presentations") %>% -->
<!--   arrange(desc(Start)) %>%  -->
<!--   detailed_entries(what  = What, -->
<!--                    when  = When, -->
<!--                    with  = With, -->
<!--                    where = Where) -->
<!-- ``` -->


<!-- ## Posters -->

<!-- 1. *Combining transcriptional and post-transcriptional regulation to predict mutations altering the gene regulatory program in cancer cells*. ISM/ECCB, 2019. (doi: https://f1000research.com/posters/8-1286) -->

<!-- 2. *Characterization of mutations that dysregulate driver microRNAs in cancer*. 4th anual NORBIS conference, 2018. -->

<!-- 3. *Clustering and redundancy reduction of transcription factor binding motifs*.  1st Student Symposium on Computational Genomics, 2016. -->

<!-- 4. *Clustering and redundancy reduction of transcription factor binding motifs*. 15th ECCB (doi: 10.7490/f1000research.1113008.1), 2016. -->

<!-- 5. *Comparing and clustering multiple collections of DNA motifs using RSAT* 12th BC2 (doi: 10.7490/f1000research.1111391.1), 2015. -->

<!-- 6. *Comparing, clustering and aligning Transcription Factor Binding Motifs with RSAT*. 13th ECCB. (**Castro J**, Thomas-Chollier M, Thieffry D and van Helden J Comparing, clustering and aligning transcription factor binding motifs with RSAT. F1000Posters 2014,5:1845 (poster)), 2014. -->

<!-- 7. *Novel computational predictions of regulons based on the observed autoregulation of the network*. 5th IECA Conference 2011. Gene Regulatory Networks -->
<!-- in the Enterobacteriaceae, 2011. -->

\hspace{1cm}

# Personalia

```{r personalia, inclue=FALSE}
jobs %>% 
  dplyr::filter(Tag == "Personalia") %>% 
  #arrange(desc(Start)) %>% 
  detailed_entries(what  = What,
                   with  = With)
```


\pagebreak
\hspace{1cm}
# References

```{r References}
jobs %>% 
  dplyr::filter(Tag == "References") %>% 
  detailed_entries(with  = With,
                   what  = What,
                   why   = Why,
                   where = Where)
```

